src/ucp/rndv/rndv.c

```diff
@@ -515,10 +515,12 @@ static ucp_lane_index_t ucp_rndv_zcopy_get_lane(ucp_request_t *rndv_req,
 
 static void ucp_rndv_zcopy_next_lane(ucp_request_t *rndv_req)
 {
-    rndv_req->send.lanes_map_avail    &= rndv_req->send.lanes_map_avail - 1;
-    if (!rndv_req->send.lanes_map_avail) {
-        rndv_req->send.lanes_map_avail = rndv_req->send.rndv.zcopy.lanes_map_all;
-    }
+    ucp_lane_map_t lanes_map_all = rndv_req->send.rndv.zcopy.lanes_map_all;
+    ucp_lane_map_t lane_map;
```

Comment:
merge with line below

Reply:
It will cross 80 characters line length limit then